environment-variables.md

```diff
@@ -6,7 +6,7 @@
 
 | Name                  | Description                                                         | Default               |
 | :-------------------- | :------------------------------------------------------------------ | :-------------------- |
-| API_BASE_URL          | Base URL for the backend API server                                 | http://localhost:5055 |
+| API_BASE_URL          | Base URL for the backend API server                                 | http://localhost:5090 |
```

Comment:
Could you undo this change/move it to the chore PR?

Reply:
Oops, reverted it to the wrong value